### PR TITLE
CAMEL-11541 Add sendVenue and stopMessageLiveLocation support

### PR DESCRIPTION
**Pull request by DenisIstomin**

### CODE REVIEW

File: components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java [hunk 1]
```diff
@@ -51,14 +48,6 @@ public void setLivePeriod(Integer livePeriod) {
...
     }
 
-    public boolean isDisableNotification() {
-        return disableNotification;
-    }
-
-    public void setDisableNotification(boolean disableNotification) {
-        this.disableNotification = disableNotification;
-    }
-
```
Comment (oscerd) [inline components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java]: Is there a reason to remove this field?

**Comment by DenisIstomin** [comment]
Updated documentation

File: components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java (hunk 1 above)
Comment (DenisIstomin) [reply components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java]: That field was added by mistake recently.
That fiels is inherited from: https://github.com/apache/camel/blob/0e0aa95517d117ba6f12504a40dc61492f89e988/components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/SendLocationMessage.java#L21

https://github.com/apache/camel/blob/d0e04616cebee03ecdd5ed5333ed57a93237b62c/components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/OutgoingMessage.java#L34

**Comment by oscerd** [comment]
Thanks, this has been merged on master. Is there still work to do on CAMEL-11541?

**Comment by DenisIstomin** [comment]
Yes, there still work to do. Made one more PR #2764 with example project improvements.
And improvements with keyboard markups are needeed.

**Pull request closed without merging**